lib/src/HttpAppFrameworkImpl.cc

```diff
@@ -1074,3 +1074,9 @@ std::vector<trantor::InetAddress> HttpAppFrameworkImpl::getListeners() const
 {
     return listenerManagerPtr_->getListeners();
 }
+HttpAppFramework &HttpAppFrameworkImpl::setDefaultHandler(
+    DefaultHandler handler)
```

Comment:
@an-tao Nit pick, but should we make `handler` a rvalue reference? Or provide both by-rvalue-reference and by const-lvalue-reference versions. It shouldn't matter overall, but copying std::function by default sounds wrong to me.

Reply:
passing by value and moving it is an idiom, it has more one ‘move’ operation than r and l reference versions, so I think it is ok, @marty1885